netlify.toml

```diff
@@ -0,0 +1,3 @@
+[[redirects]]
```

Comment:
I guess we're OK no longer redirecting all kinds of legacy links aye? They've had permanent redirects for a long time now: https://github.com/silverstripe/doc.silverstripe.org/blob/master/.htaccess

Should we continue to redirect `en/3/contributing` to the latest release? https://github.com/silverstripe/doc.silverstripe.org/blob/master/.htaccess#L46

The `/installation/*` redirects were added pretty recently, maybe we can keep them? https://github.com/silverstripe/doc.silverstripe.org/commit/f5932a0438957269a32fa7db800efe83f20fa752#diff-8052c42ab3b8aa06a3f5f788a4ddccc2

Reply:
Yes, my thinking was that the 301s have been out there long enough that we shouldn't continue to carry them around.

Have replaced the installation redirects.